From 1f1f4432f6fa9a0ed510caeb87772786b3b6d02c Mon Sep 17 00:00:00 2001
From: Owen W. Taylor <otaylor@fishsoup.net>
Date: Tue, 23 Feb 2010 16:52:55 +0000
Subject: Use gjs_context_get_native_context()

When exported into a public header, gjs_context_get_context() was
renamed to gjs_context_get_native_context(). Adapt, and remove
local prototype.

https://bugzilla.gnome.org/show_bug.cgi?id=610845
---
diff --git a/src/shell-global.c b/src/shell-global.c
index 54d4387..e3e1777 100644
--- a/src/shell-global.c
+++ b/src/shell-global.c
@@ -593,12 +593,6 @@ shell_global_display_is_grabbed (ShellGlobal *global)
   return meta_display_get_grab_op (display) != META_GRAB_OP_NONE;
 }
 
-/* Defining this here for now, see
- * https://bugzilla.gnome.org/show_bug.cgi?id=604075
- * for upstreaming status.
- */
-JSContext * gjs_context_get_context (GjsContext *context);
-
 /**
  * shell_global_add_extension_importer:
  * @target_object_script: JavaScript code evaluating to a target object
@@ -621,7 +615,7 @@ shell_global_add_extension_importer (ShellGlobal *global,
 {
   jsval target_object;
   JSObject *importer;
-  JSContext *context = gjs_context_get_context (global->js_context);
+  JSContext *context = gjs_context_get_native_context (global->js_context);
   char *search_path[2] = { 0, 0 };
 
   // This is a bit of a hack; ideally we'd be able to pass our target
@@ -799,7 +793,7 @@ shell_global_reexec_self (ShellGlobal *global)
 void
 shell_global_gc (ShellGlobal *global)
 {
-  JSContext *context = gjs_context_get_context (global->js_context);
+  JSContext *context = gjs_context_get_native_context (global->js_context);
 
   JS_GC (context);
 }
--
cgit v0.8.3.1
